crates/cxx-qt-gen/src/naming/cpp.rs

```diff
@@ -170,20 +188,17 @@ pub(crate) fn syn_type_to_cpp_type(ty: &Type, type_names: &TypeNames) -> Result<
             }
         }
         Type::Tuple(tuple) if tuple.elems.is_empty() => Ok("void".to_string()),
-        _others => Err(Error::new(
-            ty.span(),
-            format!("Unsupported type: {_others:?}"),
-        )),
+        _others => Err(err_unsupported_type(ty)),
     }
 }
 
 /// Convert any generic arguments to C++, eg A and B in Ty<A, B>
 fn generic_argument_to_string(generic: &GenericArgument, type_names: &TypeNames) -> Result<String> {
     match generic {
         GenericArgument::Type(ty) => syn_type_to_cpp_type(ty, type_names),
-        _others => Err(Error::new(
+        other => Err(Error::new(
```

Comment:
```suggestion
        _other => Err(Error::new(
```
This seems to be the reason why CI is failing.
I think you used this in an intermediate commit, but don't anymore, so it needs the `_` back.